### PR TITLE
Clarify README

### PR DESCRIPTION
**Pull request by agirdler-lumoslabs**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -28,24 +28,24 @@ Each happy hour happens at a different time and has its own special rules.
...
 
-* YOUR MISSION:
+* PRIMARY MISSION
 
 Could you look at the source code and refactor it?
 It works well enough (all of the specs pass), but the robot's AI says that it feels dirty inside.
+There is no need to keep the existing code, as long as tests pass you can safely assume it is correct.
 
-* STRETCH GOAL
+* SECONDARY MISSION
```
Comment (agirdler-lumoslabs) [inline README.org:45]: I actually feel like anyone senior should be able to get to this point with a working solution so I turned it from a stretch goal into the secondary mission(as they should probably be thinking ahead when writing their implementation for the primary mission)

File: README.org [hunk 2]
```diff
@@ -28,24 +28,24 @@ Each happy hour happens at a different time and has its own special rules.
...
-If you have time, could you also implement this extra happy hour?
+Now that you have successfully updated our robot, we have an additional task for you.
+Our juice bar would like to run a new type of promotion which is valid for the first 30 mintes of every even hour.
+During this promotion customers can purchase two mango drinks for $10
 
-** All day each even hour between HH:00 and HH:30
-Two mango drinks (Mangohattan or Mangotini), for $10.00.
-Not compatible with other promos.
-In case of collission with another promo, the other promo takes precedence.
+- Two mango drinks (Mangohattan or Mangotini), for $10.00. :: All day each even hour between HH:00 and HH:30
```
Comment (agirdler-lumoslabs) [inline README.org:51]: I feel like there is a better way to word the `when` part of this happy hour.

Comment (PolRod) [reply README.org:51]: I think it's good. Maybe the alliteration of the "e" in "each even" makes it a bit visually confusing when read.
"All day, only at even hours, between HH:00 and HH:30" could be a workaround.

If not it's fine the way it is!

**Review by PolRod — APPROVED** [review]

File: README.org (hunk 2 above)
Comment (PolRod) [reply README.org:51]: Also we could just add `Eg. 00:00 to 00:30, 02:00 to 02:30,  04:00 to 04:30, etc.`

**Pull request merged**